scalafix-tests/src/test/scala/fix/RuleSuite.scala

```diff
@@ -0,0 +1,7 @@
+package fix
+
+import scalafix.testkit.SemanticRuleSuite
+
+class RuleSuite extends SemanticRuleSuite() {
+  runAllTests()
+}
```

Comment:
Are the inputs and outputs used for anything but tests?  I'm surprised they don't live in directories here.  I still need to RTFM on scalafix.

Reply:
Correct they are only used for tests. I think they need to be in separate packages due to how semanticdb works, but I can try (maybe after this PR) to see if they work as one package.

Reply:
Ah, that makes a lot of sense.